### DISCUSSION TITLE
Make search a filter in each list of people/companies/oppties

### DISCUSSION THREAD

**Discussion by pombredanne** [twentyhq/twenty#4803] [Ideas]
I would like to search in a list of people/companies/oppties in place and the list would be filtered to the found rows
Short of this it is impossible to navigate in larger lists efficiently across people or oppties of a company. Search as it is today is not super useful.

The Filter/Field/Contain approach is three clicks too many and not an efficient approach.

**Comment by charlesBochet** [comment]
@pombredanne We already have a Command Menu including a searchBar that you can trigger through Cmd+K (or click on Search in the Navbar). Does it solve your need?

**Comment by arnavsaxena17** [comment]
Command bar is being used to navigate to a specific person/ company/ opportunity. The existing table serves well along the "process management" feature of a CRM.

For the outreach functionality of the CRM, the table filters and table functionality will need to be upgraded along the following use cases to do granular filtration:
- User will want to filter/ sort companies in-place based on a combination of "string contains", "string does not contain", less than, more than, sort ascending/ descending and so on. 
- The ability to copy rows into new tables as we want to build outreach lists comprising of multiple search criteria.

**Comment by charlesBochet** [comment]
@Bonapara, I know you are thinking about these topics, maybe you can give some clues about the vision here :)

**Comment by Bonapara** [comment]
We'll consider this going forward. As there's no ETA yet, let's continue this as a discussion!
